.ci/README.md

```diff
@@ -0,0 +1,179 @@
+# CI documentation
+
+This document pretends to clarify the CI status and configuration for the RHBA set of projects. [See projects dependency tree](https://github.com/kiegroup/droolsjbpm-build-bootstrap#kiegroup-project-structure).
+
+Table of content
+----------------
+
+* **[We use cross-repo pull requests as CI strategy](#we-use-cross-repo-pull-requests-as-ci-strategy)**
+
+* **[Where are the jobs defined?](#where-are-the-jobs-defined)**
+
+* **[Where can I find the commands which are going to be executed?](#where-can-i-find-the-commands-which-are-going-to-be-executed)**
+
+* **[How are the projects related between them?](#how-are-the-projects-related-between-them)**
+
+* **[Productization Nightly Job](#productization-nightly-job)**
+
+## We use cross-repo pull requests as CI strategy
+
+How can we assure one specific pull request will work with the latest changes from/in the dependant/dependency projects and it won't break something? This is what we call cross-repo pull requests and [build-chain](https://github.com/kiegroup/github-action-build-chain) is the way we have to solve it.
+
+Imagine a CI with more than 20 projects, different branch involves there, different product versions 7.x, 8.x ... and all of them have to be maintained for different build systems or automation services...  What does happen whenever a developer opens a set of pull requests and we have to assure they are not breaking anything from the rest of the related projects? It would be very hard to make it works and to maintain it, right? Well it is, but we have mitigated the complexity by using build-chain tool in order to delegate how to deal with it and we just care about how the projects are related between them (project-dependencies.yml file) and what to execute per project (project-definition.yml file). More information at [Cross-repo Pull Requests? build-chain tool to the rescue!](https://blog.kie.org/2021/07/cross-repo-pull-requests-build-chain-tool-to-the-rescue.html)
```

Comment:
There is written "...and what to execute per project (project-definition.yml file)..." - I don't see project-definition.yml file in the repository. Is this supposed to be a link to the *-config.yml files? I see there are e.g. Maven run options etc. for projects.

Reply:
you are right @baldimir links added for those files